book/src/user/shielded-scan.md

```diff
@@ -24,6 +24,12 @@ install the latest release:
 cargo install --features shielded-scan --locked --git https://github.com/ZcashFoundation/zebra zebrad
 ```
 
+> [!NOTE] Enabling the `shielded-scan` feature will make Zebra use the
```

Comment:
Unfortunately this not only happen when you have the shielded-scan feature enabled but when building zebra without ant flag too.

Reply:
Fixed in [7594ccf](https://github.com/ZcashFoundation/zebra/pull/8315/commits/7594ccfbb2a7a3c0dae1598ad28f43404827d8e1).